packages/amd-loader/README.md

```diff
@@ -0,0 +1,5 @@
+[![NPM version](http://img.shields.io/npm/v/@polymer/amd-loader.svg)](https://www.npmjs.com/package/@polymer/amd-loader)
+
+# @polymer/amd-loader
+
+A JavaScript library that loads AMD-style modules.
```

Comment:
Let's add some text saying that it loads path style modules and not name style modules, and that it's intended to mimic the semantics of es6 modules

Reply:
Yes, let's call out that it's a subset and mention the differences you have in the PR description.

Reply:
Just pushed a much more complete README, PTAL.